community/examples/hpc-slurm-legacy-sharedvpc.yaml

```diff
@@ -55,46 +55,52 @@ deployment_groups:
       local_mount: /home
       connect_mode: PRIVATE_SERVICE_ACCESS
 
-  # This debug_partition will work out of the box without requesting additional GCP quota.
+  - id: debug_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network1]
+    settings:
+      node_count_dynamic_max: 4
+      machine_type: n2-standard-2
+      enable_placement: false  # the default is: true
+      disable_public_ips: false
```

Comment:
Why did you need to enable public ips ?

Reply:
Removed.

Reply:
During manual testing I realized it seems it cannot ping DNS server if public ip is disabled.

Reply:
@harshthakkar01, is this new to V6 or did V5 also have public IPs and the default just changed?

Reply:
Previously, it was using V4 and all I can see is the default is to enable public ip for compute nodes.
https://github.com/SchedMD/slurm-gcp/blob/v4/tf/modules/compute/io.tf#L35

Reply:
While that was the default in the slurm repo, our wrapper would overwrite that value to disable public ip's by default: https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/fcdc5e5e484f2a2cea86a7ff9f13a9bdea969ee5/community/modules/scheduler/SchedMD-slurm-on-gcp-controller/variables.tf#L102-L105  

It seems to me like this is some sort of regression.

Reply:
Followed up on this one. We need to set up NAT and IAP first if external IP is disabled. Also added these instruction in documentation.